Small fix on URI requirement and added a paragraph explaining the issue with iso 639

fsasaki opened the pull request:
<!--
    This comment and the below content is programatically generated.
    You may add a comma-separated list of anchors you'd like a
    direct link to below (e.g. #idl-serializers, #idl-sequence):

    Don't remove this comment or modify anything below this line.
    If you don't want a preview generated for this pull request,
    just replace the whole of this comment's content by "no preview"
    and remove what's below.
-->
***
<a href="https://pr-preview.s3.amazonaws.com/fsasaki/ltli/pull/11.html" title="Last updated on Jul 9, 2020, 4:11 PM UTC (a12c9d5)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/ltli/11/e3ff7ef...fsasaki:a12c9d5.html" title="Last updated on Jul 9, 2020, 4:11 PM UTC (a12c9d5)">Diff</a>

fsasaki committed:
Merge pull request #1 from w3c/gh-pages

Merge pull request #10 from fsasaki/gh-pages

fsasaki committed:
modified requirements for URIs slightly (based on Richard's comments) and added a paragraph to explain the issue with URIs and ISO 639.

aphillips commented on index.html:
```diff
@@ -262,7 +262,9 @@ <h2>Best Practices and Recommendations</h2>
-		<p class="advisement" id="ltli-language-information-in-uris-req"><a href="#ltli-language-information-in-uris-req" class="self">&#x200B;</a>Applications (e.g. in the realm of RDF) that provide language information as part of URIs SHOULD use [[BCP47]].</p>
+		<p class="advisement" id="ltli-language-information-in-uris-req"><a href="#ltli-language-information-in-uris-req" class="self">&#x200B;</a>Applications that provide language information as part of URIs (e.g. in the realm of RDF) SHOULD use [[BCP47]].</p>
+		
+		    <p>Currently, URIs expressing language information often use values from parts of ISO 639. This leads to situations in which there are ambiguities about what the proper value should be, e.g. for German "ger" from ISO 639-1 or "de" from ISO 639-2. By using BCP 47 and its language sub tag registry, such ambiguities can be avoided, e.g. for German, the registry contains only “de”.</p>
```
ISO 639-1 == alpha2 and 639-2 == alpha3, so the example is backwards. 

Use &lt;code&gt; around language tags (remove quotes).

fsasaki replied:
Thanks, fixed.

fsasaki committed:
Fixing "language tags and URI" paragraph:

Based on Addison's comment https://github.com/w3c/ltli/pull/11#discussion_r452322605
